setup.py

```diff
@@ -233,6 +233,7 @@ def do_setup():
             'simplejson>=3.2',
             'tables',
             'uncertainties',
+            'llvmlite<=0.30.0',
```

Comment:
please add a short comment as to why this version limit is imposed, so we are better aware when other things whether or not we can relax this restriction (or if it continues to be necessary)